src/controllers/helloCommand.ts

```diff
@@ -0,0 +1,5 @@
+import { DISCORD_RESPONSE } from "../constants/responses";
+
+export function helloCommand(userId: number) {
```

Comment:
type for return type of function is missing (add wherever it is missing)

Reply:
Will do this